pandas/core/generic.py

```diff
@@ -10932,6 +10932,12 @@ def _add_numeric_operations(cls):
         """
         axis_descr, name1, name2 = _doc_params(cls)
 
+        @deprecate_nonkeyword_arguments(
+            version=None,
+            allowed_args=["self"],
+            stacklevel=find_stack_level(),
```

Comment:
ah this probably needs to be
```suggestion
            stacklevel=find_stack_level() - 1,
```
 because of the whole `_add_numeric_operations` and `setattr(cls, "any", any)` thing